### ISSUE TITLE
6 GPU miner freezing windows. Runs fine for a while then stops?

### ISSUE THREAD

**Issue by hungryhungarian** [nicehash/NiceHashMiner-Archived#1085]
Im running 6 GTX 1080 ti ZOTAC AMP Extremes. Four GPU's run like a charm with no issues. Five I havent tested extensively but when I did it seemed fine. I will do more test soon and update. However with all six, stability issues or something of the sort come up. It will run Nicehash for a good 30-45 minutes and do really well, and then all of the sudden either freeze windows or the prompt will repeat "unknown error" over and over. 

Ive done everything I can think of and stuff that Ive found online. ANY help would be greatly appreciated. Ill even send you eth or something if you help me resolve this frustrating problem.

**Comment by mritten** [comment]
Having the same issue here, have 4 GTX 1060's and 1 GTX 1070, windows 10 is up to date and using the most recent nvidia drivers.  Nicehash will run daggerhasimoto for a few hours and then windows will restart or will lock up.  I'm going to re-install windows 10 and start over from scratch to see if that fixes the issue for me.  All benchmarks work fine.  My other system with just 2 1060 Zotac's mine daggerhashimoto all day long without any issues.

**Comment by hungryhungarian** [comment]
I actually made a small breakthrough today. I have been testing only using 5 GPU and it was doing the exact same thing as when I used 6. I unplugged and put the PCIe risers in a different order on the motherboard, and now the 5 GPU setup has been running flawlessly. No idea if this is a fluke or what but I'm hoping when I plug the 6th in it'll run.

**Comment by mritten** [comment]
I wonder if your issue may be related to a riser being bad?  Also what is your bios PCIE Gen set at?  I changed mine to Gen 2 and I still had the issue of system restarting or freezing.

**Comment by hungryhungarian** [comment]
I used the same risers, but in a different order so it shouldn't be that. I have mine on gen 1. Also I should not that I put the HDMI out put on another card that switched pcie spots with the original. If I'm not mistaken this actually solved some power fluctuation issues with the first card.

**Comment by mritten** [comment]
What size PS are you using?

**Comment by hungryhungarian** [comment]
Using 2 psu just to be safe, one 1600w powering the motherboard and 4 GPU's. And one 850w powering the other 2 psu's. It's overkill but just barely.

**Comment by mritten** [comment]
Ok just wondering if it was somehow under powered and that was the issue.

**Comment by mritten** [comment]
A fix that I found to work is to increase the windows 10 virtual memory system size.  It defaults to a low amount.  If you increase it to an 8gb size of ram, I set it to initial: 8,192 and maximum: 12,288.  Nicehash utilizes the virtual memory and can cause issues from what i've experienced if not updated.

**Comment by anonymoous** [comment]
I am having the same problem.  It runs for a few hours then stops.  I also have 6 1080's

My virtual memory is set to like 50gb so that's not an issue.

My PSU is a 1600w

**Comment by anonymoous** [comment]
Could this be a heat issue?  Are you guys overclocking?

**Comment by hungryhungarian** [comment]
It definitely isn't heat for me, mine are running between 40° and 60° never over

**Comment by hungryhungarian** [comment]
I increased my virtual memory as well before posting this so it's not that, at least for me

**Comment by mritten** [comment]
I switched over to Claymore miner which seems to be more stable so far especially with my AMD and Nvidia card rig.  I will know more after I have it run for a day or so, but so far the hashrate is staying stable where in Nicehash the ETH miner hashrate fluctuates and the miner would restart periodically.

**Comment by anonymoous** [comment]
Yeah i switched also because this Nicehash miner is just un-usable for me at the moment.  Using ethminer

**Comment by anonymoous** [comment]
Do the devs read this? Would be nice to get a comment from them on this issue

**Comment by theLosers106** [comment]
Two things that I can suggest you should try is that split the 6 GPUs to 2 different NiceHashMiner instance (GPU1,2,3 NHM Instance 1 & GPU4,5,6 NHM Instance 2). Another thing you could try is that I had once had an issue running it for more than 1 hour and it was because of the RAM. Hopefully this helps.

**Comment by myrig** [comment]
Hi there i had the same problem with gtx 1070 any time that you put more display cards it require mor memory to send to processor. So i had 5 gtx 1070 than anytime that has big transactions to confirm your rig it crashes the system. So i solved by puting 4 more gb on motherboard and everything works fine hope this will solve your problems. Simply if you have 4gb increase it.

**Comment by hungryhungarian** [comment]
Update guys, I had ordered 2 8gb ram sticks for my personal PC and decided to try those out on the rig. It's been running for 3 days straight now. Still VERY glitchy, so once it started running I just straight unplugged any mouse/keyboard input and let it do it's thing lol. I check that it's still running on the nicehash website with MyMiner.

**Comment by p1r473** [comment]
So you just needed more RAM?

**Comment by hungryhungarian** [comment]
That was one of the problems, man to be honest I can't list the amount of things I tried. It just started working all of the sudden but yes try adding RAM

**Comment by mritten** [comment]
Just to confirm you already increased your virtual memory size which did not work for you?

**Comment by p1r473** [comment]
I am having serious stability issues.
I am getting a lot of freezes (have to reset the power) and some TDR BSODs (I already increased TdrDelay to 60sec) and I am on the latest drivers
I am hoping someone can help me. I just built a 6x 1080 TI rig (aorus gigabyte) and I am not using their software. The return window on my GPUs is closing fast, and I need to get this stable ASAP!

Yesterday I mined for several hours stable, so I don't think its a hardware issue.

Why do I keep freezing and BSODing??

I just ordered 2x 8gb ram (was using 2x4gb ram) as per your recommendation. What else do I try? Desperate to get stable! Was a huge investment for me, and it crashes every hour!!

Any more potential fixes I should try?

**Comment by hungryhungarian** [comment]
Yes I had already increased Virtual memory, no help at least for me. 

I never got a BSOD so I can't help you there. Keep googling and try everything you find. Eventually you'll find the problem. I'll go through my bios when I'm home and share all my settings.

**Comment by p1r473** [comment]
Its mostly freezing, but some TDR BSOD too

**Comment by mritten** [comment]
Please review the previous comments on here.  
What Power Supply do you have?  
Fresh install of windows?
Is windows up-to-date?
Does your motherboard have a molex input near the PCIE slots?  Sometimes a motherboard requires this to be plugged in when utilizing multiple cards.
BIOS is up-to-date?
Try PCIE Gen 1, Gen 2 Settings
Latest Nvidia drivers only, do not install VR and other misc drivers.
Disable motherboard audio as well as Nvidia sound.
Virtual Memory size increased.
Update windows power settings and ensure PCI is set on maximum power and disable other power save settings (Hybrid, Hard drive shut down)
Mine on stock settings at first to ensure system is stable before beginning to overclock.

**Comment by p1r473** [comment]
Up to date fresh install of Windows.
2x 1250w psu.
I did not plux the molex for the motherboard as I am using powered risers
Bios is current. PCIE gen 1 and 2 tested
Nvidia drivers only installed, no other software
Virtual memory increased a lot
Power settings max
Overclock is only +0
Nvidia hdmi sound output disabled

Will try to disable mobo audio.... Only thing I havent tried yet

**Comment by hungryhungarian** [comment]
This may sound weird or pointless but try rearranging the order that the risers are plugged in. For some reason one of my GPU's really didn't like being the vid output and had large power fluctuations. Once I switched to another as output both of them had stable power usage.

**Comment by p1r473** [comment]
Will try that too thanks!

**Comment by mritten** [comment]
That is a good thought, or just try plugging in 1 card first and getting it stable, then add another card and so on until eventually you have all cards running.  If it becomes unstable at any point when adding a card then you will be able to figure out if it's an issue with the riser on that card or something else.

**Comment by p1r473** [comment]
I also heard running 2 instances of NiceHash may work (3 GPU per instance)
Will try this all....

**Comment by p1r473** [comment]
Also will try to disable non-profitable algorithms

**Comment by myrig** [comment]
Hi there.
I am having many trubles with stability and now trying those options and lets hope if this solve the problem by the way i will keep updating my experience with you guys :)
I have GTX 1070 1x6 pins & 1x8pins power consumption connectors
1. No more than 4 gpus per rig. 
2. Incerease virtual memory to mycomputer/properties/advanced/virtual memory to atleast min & max 32000 mb.
3. Keep the PSU temperature low. most of  the troubles is cousing the power feed.

Hope this soleve your and my problems.

**Comment by mritten** [comment]
Make sure you only are testing stability with 1 algorithm.  Another way to test stability is to just download the latest Claymore 9.6 and set it to mine on the nicehash pool if that's what you prefer.  This way you can monitor temps and see if the crashing is because of the nicehash software.

**Comment by myrig** [comment]
Guys i found a solution resatarting the rig atleast every 6 hours before the system crashes also new version of nicehash is better than old one. Find wich algorythm is causing the crash and disable them. I left only daggerhashimoto. Also it pays better than others.

**Comment by mritten** [comment]
Have you tried another miner out?  Like claymore to see if it still crashes.

**Comment by fadaor** [comment]
hola a todos, por favor traducir.
tuve en cuenta todos sus comentarios debido a que tenía el mismo problema y lo resolví gracias a ustedes.
mi pc es el siguiente: core i3 7 generación, placa madre msi z270-a pro, psu gamemax 1050 w, 6 gpu 1050 ti.
resulta que msi creó nuevos drivers para placas madre especiales para minería, lo actualicé y fincionó por unos días, hasta que continuó el inconveniente de bloquearse al minar con 6 gpu, pero con 4 pgu funcionaba normal.
como explicaron ustedes, cree tdrdelay con valor 10, aumenté la memoria ram min. 8 max. 12 e inicié dos veces nicehash y repartí las gpu ( 3 y 3).
van 6 horas de minería sin complicaciones, ahora voy a probar overclocking si se mantienen estables las gpu.

**Comment by smartmanjp** [comment]
There any fix for it yet i think everyone have the some problem i 4 Rigs 32 Cards all the 4 crash to many time every day i try everything i build computer all my life I know a lot of computers windows and Linux but i never see an systems crash so many time and i think there no fix for that if anyone have any way to to fix it please let me know

**Comment by pstealth** [comment]
Possible solution....I have the same issu with 6 Zotac 1080ti amp extreme...I found that one of my card on MSI AFTERBURNER was set to power limit 120. This card was causing the crash...All others card was at power limit 100....So check your power limit. The Zotac are already push at max stock. When I have install the last card, I have forgot to CUT the link between power limit and temp in msi afterburner for that last card only...So when I have push all cards to temps max 90 degres, the las card raise thepower limit to 120 by default.

**Comment by mritten** [comment]
Well for now Nicehash is down so you should be using a separate miner
anyways.

**Comment by pstealth** [comment]
I use claymore now...mining Zcash and money send every hours in my trezor wallet, learn my lesson. (I loose 1 day of mining with Nicehash, 200$) I'm lucky..I have made a withdraw of 800$ one day before de Nicehash Hack.

**Comment by minorminer59** [comment]
I am new to mining and having all of these issues. Please advise if I need to create a new thread. 

I am running 6 Evga gtx 1060 6gb SSC ( cannot find a stable OC settings.) 3 cards are samsung and 3 are Hynix. Two PSUs 850w and 1000w.

I was able to get 5 cards semi stable running 65w + +50core +500mem after about two days it would freeze the whole pc. Have to hard reboot.

I added a 6th card and cannot run for more than 5 minutes without it freezing. 

I removed 6th card and still freezes with only 5. 

I am using leboo risers. Is it possible there are issues with dual psus or maybe bad risers? 

Where do I begin to trouble shoot this mess?

**Comment by mritten** [comment]
How many risers do you have plugged in to the same strand from the power
supply?  I found that anymore than 3 risers plugged into the same sata
power cable from the power supply can create issues.  To help narrow your
issues remove the overlock and see how long the system will run before
locking up.  What OS are you using?

**Comment by minorminer59** [comment]
I have 4 risers on the 850w and 2 on the 1000w (the 1000 is new.)

I’m running Windows 10 Pro.

**Comment by minorminer59** [comment]
Also main board is MSI Z270 Gaming Plus with i7 -7700k 
3600 - 32 GB ram. not sure if info was needed

**Comment by myrig** [comment]
Hi there i saw many threads about this problem. 
First increase virtual power. This will solve freezing problem.
Seccond make a bat file to force your pc restart every 2 hours and put it on startup folder. Do the same by creating a shortcut of nicehash/claymore/cgminer or what softwer do you use. This is to start apps at startup of pc so you dont have to take care always turning them on.

Best oc settings for gtx1070 tem 70c 145 and the last 500-600 at afterburn depand on script that you load. 

Hope this helps you guys

**Comment by myrig** [comment]
Hi there i saw many threads about this problem. 
First increase virtual power. This will solve freezing problem.
Seccond make a bat file to force your pc restart every 2 hours and put it on startup folder. Do the same by creating a shortcut of nicehash/claymore/cgminer or what softwer do you use. This is to start apps at startup of pc so you dont have to take care always turning them on.

Best oc settings for gtx1070 tem 70c 145 and the last 500-600 at afterburn depand on script that you load. 

Hope this helps you guys

**Comment by minorminer59** [comment]
My Virtual memory is set to 16gb I’m running claymore eth miner. The settings for 1070 work for 1060 6gB SSC? Why would I need to restart every two hours? I mean is this really necessary?

**Comment by myrig** [comment]
Depand to miner. Restart gives the psu and vga a little time to cool for a while. Most of times the power connectors get burned and sometimes affect the gpu. I have 6 months runing 24/7 no problem yet

**Comment by mritten** [comment]
Distribute the risers so that 3 risers and video cards are running off the
850w and 3 riser and video cards are running off of the 1000w.  Try that
out and make sure you have the latest driver and windows updates.

**Comment by minorminer59** [comment]
Ok thank you for the tips. If three risers per psu are recommended, how do people build 8 Gpu rigs? Just curious not doubting. Also, would a bad riser have anything to do with this? Or would a bad riser not work at all?

**Comment by mritten** [comment]
A bad riser could do it as well.  Usually I will run 2 or more sata cables
to connect 3 risers on a single cable each, but since you have 2 Power
Supplies it's worth distributing the load to rule out any other issues.

**Comment by mritten** [comment]
Also some people ditch the sata to 6 pin connection going into the riser
and just use 6 pins off of their power supply if they have enough to spare.

**Comment by minorminer59** [comment]
You don’t need to power the riser?

**Comment by mritten** [comment]
No they use a 4 pin directly off their power supply instead of the sata
connector.

**Comment by minorminer59** [comment]
Yes my risers are 6 pin to sata and my gpu is an 8 pin. So I can ditch the riser power?

**Comment by minorminer59** [comment]
Update- with 6 cards running at 100% power and no settings changes it seems to be running stable. However this isn’t great either 3 cards are doing 19 MHs and 3 are doing 15. 

What are some next step trouble shooting? Also as a note windows seems to have a bit of trouble booting with 5 cards. If I swap pci slots or boot with 5 cards then reboot with the 6th it seems to work.

**Comment by ghost** [comment]
Hey guys,

so I had exactly the same issues and I think I fixed it. Since your comments helped me out a lot, I wanted to write what I did so far. Here is my system specs:

Asrock Z270 Pro4
PSU bequiet 1000W
6x Gigabyte GTX 1060 3GB
Intel Core i5-7500 CPU Prozessor, 4x 3.40GHz
Ballistix Sport LT 8GB DDR4-2400 CL16 rot SR

Before somebody is asking: I also use this setup for gaming ;)

I was using XMR_Miner with 36 blocks and 16 threads, which works usually good for gtx 1060 3gb.

What happened: I was setting up the system for the first time. I was using 4/5 PCIe and 2/2 M.2 interfaces for the 6 GTX 1060. It worked perfectly for one day. After I was forced to install Windows 10 1709, system was crashing all the time and screen frozen after about 2 minutes of mining. Then I rolled back to the last windows update, but somehow there still needs to be a driver issue that I cannot resolve anymore (even installed new drivers for cards), since windows now installs 1709 be default (even when creating a boot USB). This somehow results in a conflict of drivers, since using only one M2 interface works perfectly!!

How did I resolve the issue: I discovered that one of the cards was having about -5% hashrate compared to the other cards. So I decided to switch the cards around since there is one spot free on the mobo. I am using now 5/5 PCIes and 1/2 M2 interfaces. Fortunately (fingers crossed) nothing happened for about 24 hours, while having crashes in time frames of minutes before. The M2-PCIe adapter works fine since I switched them in order to verify this. At least to my knowledge, it has to do something with the Windows 10 1709 update and a resulting driver conflict for the M2 interfaces, which leads to lower performance of one of the M2s. Thanks for nothing Bill...

I can even overclock and undervolt now, without any issues.

Thanks to everybody here!!! Sluicifer is mining again :D

Greetz

**Comment by Lostinbios** [comment]
tried almost all in this topic ...i have my 1050 ti rig (5 x pgu) near a window and I experience that when  i close the window my rig does the same... could it be heat ? i mean the risers ARE cheap china crap and when capasitors of the cheapest kind gets hot, they fail...
Im testing out more heat issues today !
running succesfully for 14 hours now with the window open Copenhagen Denmark is 2 degrees celcius outside (35 farenheit)
stay tuned :-)
.......update
I think i can rule out heat... it crashed with the window open..
I then skipped the x16 port and only used x1 PCIe ports and the whole computer now runs much smoother and no crash so far 20 januar 2018

**Comment by dragon9820** [comment]
Rig:  MB: MSI Z270A SLI PLUS,  CPU: i5 7500, 8GB ram, WIndows 10 PRo, HD: 64 GB SSD, PSU: 1050W Seasonic snow silent, GPU: 4 gtx1070& 2 gtx1070 Ti
Powered Risers 3 cables to PSU Sata, Powered GPU's 6+2 pin to pcie on psu
Display is ran by 1st gpu with vga to hdmi adapter.  
BIOS settings:  PCI 0 - Gen2, PCi 1- Gen2

After benchmarking I came down and the screen was off, I believe windows had froze,  So held the button and restarted, still not screen.  I had to go into BIOS and disable the 4g crypto setting  then my screen would turn on.  But with that setting disabled only 4 cards are able to use, 2 have code 12, stating not enough slots.  Now with just 4 gpus with the 4g settings disabled  I am able to start mining for about 10 minutes before either windows freezes or reboots.   I have a fan pointing right at the rig and looking at the temps in Afterburner, they are between 50 and 65 degrees Cel.   Just curious if you guys have any thoughts?  Thanks!

**Comment by mritten** [comment]
Try running the display off the motherboard and try benchmarking first
without overclocks.  Then slightly begin to adjust the cards one at a time.

**Comment by dragon9820** [comment]
For some reason I cannot get the display to come on plugged itno the integrated graphics card once the gpu's are plugged in.  If I unplug all gpu's from the MB, then I can use the hdmi slot on MB.  Does that make since?

**Comment by mritten** [comment]
You need to go into your BIOS settings, change the primary display settings
to IGFX or onboard, whichever the settings are.  Also make sure your BIOS
is up to date.

**Comment by dragon9820** [comment]
ok i'll double check that   thanks

**Comment by killdog99x** [comment]
dragon9820  from my own experience 1070's can pull up to 200w each..your powersupply just may not be keeping up. causing the freeze

**Comment by dragon9820** [comment]
well after another long night,  still no success   I did change the display setting to IDG  that definitely seem to help my display issue.  I thought I had it at all working at one point after I disabled the cpu and just was running the 6 gpu, changed the pci settings to gen 1, nice hash ran for 20 minutes  then I checked back and had the blue screen of death.  Then i plugged them in one by one starting up and shutting down, when i got to the 4th slot, big 16 slot, display would not come on, no matter which gpu I plugged in.  So I just had the 3 plugged in to see if I could get stable.  Same issue and changing the pci settings back forth now the miner will go for a about 4-5minutes and windows just locks up.  
So I would think the psu should have no issue power 3 gpu's, 
could it be my hard drive??  its only 64gb and I've read a lot saying to crank your virtual memory
could it be too much heat, afterburner says around 60-65 degrees
pci bus clock latency: is that very sensitive, i've changed it from 32,64,96  
going through event viewer I did find an error nvlddmkm quite a few times

**Comment by banshee1971** [comment]
I build 1 RIG with 
MSI B250 PC MATE + Kingston ValueRAM 4GB (1x4GB) DDR4 2400MHz DIMM (KVR24N17S6/4) +  
Intel Celeron G3900–2,8 GHz + EVGA SuperNOVA 55 G2 850 W Fully modular
I have 3 x MSI RX 580 Gaming X 8Go and 1 MSI RX 580 ARMOR 8 GO OC + OCZ 120 Go SSD (old one i already got doing nothing at that time)
Installed with Windows 10 Home (all windows Updage) + AMD Crypto Driver + Change all BIOS to optimized it !
Running fine for 2 weeks now at 630 W ! With EtherControl (to control remotly the profile, and see in action working).


Now i try to configure a RIG for a friends, using exact same component (except the PSU are EVGA 850GQ and Samsung 250 Go SSD + Windows 10 Pro. 
3 x MSI RX 580 Gaminx X 8 Go. 

System crashing all the time. I unplug the Riser of one card, seem crashing. But if i use 1 card at a time, very stable...

So i completly unplug 1 GPU (not only the riser), and start Claymore 10 (start.bat) instead of from Ethercontrol ... seem stable now for fiew hours.

Something tells me it's the power supply that seem not capable of handle all GPU... 

Now next step, is to wait 24 hrs. If Stable, just plug the second card in the GPU connector only, and re-test it ! So valid it's power issue.

Seem that, from this thread, that all Windows freezing issue are power related most of the time.

**Comment by Alexisgarci** [comment]
Im having the same problem. I have 6 1080ti GPUS, z270 pro carbon, i7-7k, 32GB, 2 PSU's 1200W, windows 10 pro, all drives up to date and windows to. Windows freezes when i activate the 6 gpu. I can run 5 gpu's for days but when i turn the 6 onn on nicehash to mine, windows freezes after a while. 
I already changed risers, changed the order of the PCIE's, virtual memory is at 66GB at up 80GB, i tried GEN1 Gen2 in PEG01 and PEG 02. 
Anyone knows how to stop this freezes?

**Comment by mritten** [comment]
Are the cards overclocked at all?

**Comment by Alexisgarci** [comment]
Nop, i dont overclock, them, and ive tried underclock them.

**Comment by mritten** [comment]
How do you have nicehash configured?  Is it mining the most profitable
coins or do you have it set on a specific algorithm?  I would try mining
different algorithms and test each one for a few days to determine if it's
a specific one causing windows to freeze.  Another thought is to try and
plug the risers in directly with a 6 pin connector from the PSU instead of
the sata adapter to see if this makes a difference.

**Comment by dragon9820** [comment]
yes this issue is driving me nuts, still trying to figure it out, my next step is to drop down to windows 1703 version and see if that helps,  but I've tried alllll kinds of things between drivers and BIOS settings   
getminer, I have noticed when it works well for a long period of time, its mining using hashimoto  ive noticed it freezing more frequently with others, whether it be equihash or neoscrypt

**Comment by Alexisgarci** [comment]
Its connected directly from psu to the riser, no adapters.

**Comment by Yazdan-Ahmed** [comment]
I THINK I HAVE THE SOLUTION.
I was having this same problem on four different rigs, even one with just 4 gpus. MSI z270 gaming m5 MBs, 1080ti rig, 1060 6gb rig, 1050ti rig, and 1060 3gb rig.

So I redid everything from scratch, this time I installed etherminer first, started running it and then added all the other apps I use on my rigs. 

As I  was installing MSI afterburner the rig froze up. I restarted it, uninstalled afterburner, and then started etherminer again.  So far its going well for the last 15 min. used to freeze after 3 min.

**Comment by Alexisgarci** [comment]
but you dont use nicehash?

**Comment by dragon9820** [comment]
With uninstalling afterburner. Won’t you lose your overclocks?  Or not

**Comment by Alexisgarci** [comment]
yes there is another programs to use. Ill give it a try tomorrow. Every time i set all 6 to mine it gives me the Blue error of video tdr failure nvlddmkm.sys. I think i already saw every vid on youtube about it and none works :/

**Comment by Yazdan-Ahmed** [comment]
UPDATE. Its not afterburner, afterburner just starts using more CPU, and if youre not configuring the batch file exactly right for your hardware you will use too much CPU.

Run task manager while you run your miner and see if the CPU usage stays high. If it does, the problem is that your cpu is being overuseed. Search the cpu issue with your hardware to find a solution. For me, using nvidia, I had to switch the -G in etherminer to -U . Working great now, CPU staying below 20.

**Comment by banshee1971** [comment]
My cpu is a celeron, and dont go more than 10%. When windows freeze (usualy after1-2hrs) there is no clé on what happen (3 GPU app open to monitor GPU, and task monitor in performance monitor mode). The freeze screen show me normal behavior.

I got no bluescreen and no error on the event viewer.

**Comment by banshee1971** [comment]
The only différence between those 2 rig : windows pro and home edition (home edition stable with 4 GPU)

I use msi rx 580 game x 8g

**Comment by banshee1971** [comment]
Im using claymore Dual (used in ether mode only)

**Comment by Yazdan-Ahmed** [comment]
Well this seems to be working for me, of course it hasnt been running that long though. My cpu was spiking, but now its staying low. Again, check your batch file settings, make sure it matches your hardware and OS. The little letters make all the difference.

**Comment by Alexisgarci** [comment]
where do we check the batch file? i have an i7-7k and i mine with it, so its not the cpu in my case. What windows do you guys use?

**Comment by Yazdan-Ahmed** [comment]
I use i3s across the board. Windows 10. If your CPU is at 95% or more at any point, it is likely the CPU. otherwise it could be something else.

**Comment by Alexisgarci** [comment]
Windows home or pro?

**Comment by Yazdan-Ahmed** [comment]
The batch file is your start file for the app. Its in the folder with the app. Using pro.

**Comment by dragon9820** [comment]
I’m using nice hash     I’m really wondering if it’s a windows issue.  When I back it down to 1703. I’ll let you know if it still freezes every 1-2 hrs like before

**Comment by Yazdan-Ahmed** [comment]
When I was using nicehash it kept crashing, so I disabled dagger-hashimoto and that seemed to work. I see a lot of people talking about one of the NH currencies causing the crash.

**Comment by dragon9820** [comment]
Interesting.  Because that’s what algorithm Im always using.

**Comment by minorminer59** [comment]
I had all of these crashing issues too. I ended up swapping all of my risers for the latest ubit ones. I blew out all of my drivers and reinstalled. I have been running smooth for almost two months now. Zero system interupts. 

Open up your activity monitor. If you see fluctuations in your system interupts more than likely bad risers.

**Comment by dragon9820** [comment]
Well just an update here.  I’m using the latest nice hash miner version and also use legacy    Windows usually freezes within 1-3 hrs.   I went ahead and turned windows updates back on and and installed update KB4058258.   Been up and running for 11 hours straight!!!

**Comment by Alexisgarci** [comment]
I cant turn the 6 GPU on in nicehash, if i turn it on ill get the video tdr failure nvlddmkm.sys blue error after a while. Im mining with 5 gpus only :/ sucks

**Comment by dragon9820** [comment]
yea, definitely!  does your board have the 4g crypto option in BIOS?

**Comment by Alexisgarci** [comment]
yes, i already turned it on and off. You guys have Peg0 and peg1 or only peg0?

**Comment by banshee1971** [comment]
I have 4g crypto option in BIOS, and on my working Miner, i have only the PEG option enable. Working stable since i finish adding the latest GPU (was stable all the time anyway)
here the XLS sheet i create to document my BIOS config of this MSI motherboard !
![2018-02-02_13-20-35](https://user-images.githubusercontent.com/7092486/35748399-eab3c124-081b-11e8-9925-d9c67b010e2c.png)

The 4 G option only appear since the latest version of the board. I upgrade the BIOS of both RIG, and this as no effect on both ... mine still stable, the other still crashing !

On the Crashing RIG ;
I change the option between PEG and the other option (to have the onboard card on), in case it improve things, play arroung with the 4 G option (enable/disable),...

The board on both RIG are MSI B250 PC MATE
Both has the same brand /model of CPU and memory. But now my friends is ordering a 1000 W PSU just in case his 850 W are faulty, and he just receive his 4th GPU (so the RIG will have 4 instead of 3 GPU).

I heard a comment regarding ethermine working better with pair of GPU... is it possible that it's the case with Claymore ? (because my issue are with 3 GPU... 2 Works like a charm)

The other, since this faulty RIG has Windows PRO ... and i know that there is more option activate in PRO than HOME... is it possible also that it's there that reside the issue ?


I plan to perform a test this weekend... install on a spare drive Home edition just to valid this.

I'M 100% sure about 1 thing, the GPU and GPU riser works fine because i made them work strait 48 hrs without issue.

**Comment by Alexisgarci** [comment]
We have diferent issues. Mine don’t freeze I’m just getting the blue error on video_tdr. I’ll try your setting and I’ll see if I get a crash. When I get home I’ll do this and I’ll share with you my settings. This shit is getting me crazy. I think it’s my mobo I have a msi z270 pro carbon. I’ll have to try another mobo to see. I’m considering switch to Linux tho.

**Comment by mritten** [comment]
Nvidia cards should use a PCI link speed of GEN 2, AMD Cards use a PCI link
speed of GEN 1.

**Comment by Alexisgarci** [comment]
Ive tried both. I hve peg0 and peg1 ive tried gen2 in both gen1 gen3 and cant still add the 6 gpu

**Comment by mritten** [comment]
Not all motherboards fully support 6 GPU's even if they have enough PCI
slots.  It's an issue with a few motherboards that may have 6 PCI slots,
only 5 GPU's will work.  That's why the Asus Z270 boards are so popular.

**Comment by Alexisgarci** [comment]
i have a msi z270 pro carbon, ive seen already people using it with 6 gpu but seems i cant do it :/

**Comment by mritten** [comment]
Google your motherboard and you will find out that it's a common issue with
mining with 6 GPU's.  It's either hit or miss with MSI.  Have you tried
using a M.2 adapter to see if that makes any difference?

**Comment by Alexisgarci** [comment]
havent tried the m.2 adapter yet. Ye i think is that, or we are lucky to get one mobo who works or one who cant handle it :/

**Comment by dragon9820** [comment]
I'm sure you've done this but when I was experiencing issues between 5 and 6 gpu   how I solved it was, I did a clean install of windows, no gpus yet, changed my graphics to igd, saved and started windoes, I downloaded the latest NVidia driver, installed it (just Driver)and shut down. installed the 1st gpu powered on, waited about 6-7 minutes, and I noticed the screen would blink.  Once it blinked, the driver was successfully installed.  power down  go into bios turn on 4g, turned off audio, disabled serial, save settings and started windows.  wait a few minutes and power down.  Then I would add 1 gpu at a time, and each time in windows the screen would blink after 6-7 minutes.   I would just repeat for the rest.  after doing this, I had all 6 successfully installed for the first time.    I have an MSI Z270 SLI PLUS   these steps may be overkill but it worked for me!

**Comment by Alexisgarci** [comment]
well ive installed and uninstalled windows 3 or 4 times, and never installed 1 by 1. Ive never tried the windows home, but i dont think thats the problem.

**Comment by dragon9820** [comment]
here's a decent guide for you, helped me out anyway
https://www.msi.com/blog/Mining_Intel_Motherboard_Go_with_MSI_100_200_Series_Motherboard

**Comment by Alexisgarci** [comment]
tnks a lot bro, ill give it a look when i get home. ill post my bios setting anyway may help some one to stop the crash's since i got to stop them!

**Comment by banshee1971** [comment]
Wow, i did create the same documents (with no picture) on how to configure the rig. I didn't know msi already did one. Very well documented

**Comment by banshee1971** [comment]
My motherboard limitation is 4 GPU btw.

**Comment by Alexisgarci** [comment]
in that guide they dont even change the peg0, they leave it at auto. They dont much stuff in the bios.

**Comment by mritten** [comment]
I would make sure the BIOS is up to date as well.

**Comment by dragon9820** [comment]
I have to have mine set gen 1 and 32 bus clock

**Comment by dragon9820** [comment]
Alexis,  Just thought I’d give you an update.  I went ahead and started from from scratch again, just tired of my rig freezing all the time. So I installed ver. 1703.  Seeing much better stability!   Peg gen2 and bus clock at 96.  Been going strong since this morning.  Pain in the ass to do but hopefully it keeps it up. https://tb.rg-adguard.net/index.php?sid=74a7cf51-12ed-19f5-2885-3b48ad9ee67b

**Comment by Alexisgarci** [comment]
Hope it works out for you, i guess ill leave this rig with 5 gpus and get this gpu to the next one... I dont know if im going to do it all over again lol I think its my mobo problem, i think it cant handle 6 gpu, im waiting on a m.2 adapter to see if that will work.

**Comment by dragon9820** [comment]
Lol I can’t blame you  sucks starting over.  I will say, still running strong. 24hr uptime.  I’ve noticed a huge difference with this version of Windows. As soon as I installed I cancelled the updates so I could stay at 1703.  I can overclock with no issues    Been a long process for me.  I just hope I can save someone the amount of time I spent figuring this out

**Comment by microtechies** [comment]
Awesome to have found a work around but would love to find a fix from Nicehash Legacy to avoid this. I've just been using claymore myself without any problems. As soon as I use Nicehash with in 1-2hours BSoD

**Comment by dragon9820** [comment]
I’ll do another another benchmark with legacy tonight and let it run for a while. We’ll see if I have any issues.  Before it didn’t matter which miner I would use.  I’d freeze or BSOD.  So I’ll let you know what i find

**Comment by Alexisgarci** [comment]
so its working dragon?

**Comment by Xinef87** [comment]
I've been following this thread for quite bit. Completed my rig setup last week. 
MSI Z270 SLI PLUS with 4x Gigabyte GTX 1070 Windforce

I have been having freezing issues until yesterday. The only thing I changed from all the other steps I've read or seen is I uninstalled Teamviewer. I have been running over 14 hours without any issues. 

I found this out by leaving Task Manager opened for the duration of my test and as soon as I remoted into the rig, my cpu usage shot up to 99%. I removed Teamviewer, reset my clock settings to the cards stock settings and ran a precise benchmark on Nicehash. With Task Manager still opened and now using Chrome Remote Desktop, cpu usage has not turned red once. Will be playing with the clock settings today to adjust for best hashrates, but maybe this might help others. Will also be adding a 5th card soon, hopefully that goes smooth.

**Comment by Alexisgarci** [comment]
well im using the new nicehash legacy, im using all 6 gpu and its all ok for now. I formated my windows again(4 times) installed the old version before updates, ive followed msi guide and its working for now like 15H straight. Lets hope it stays good! edit: dragon9820 how its going for you? Mine is running the 6gpus for now

**Comment by dragon9820** [comment]
Sorry for the delay.  Been running legacy for over 24 hours.  Seems pretty good. No issues.  So far !!!

**Comment by Alexisgarci** [comment]
good, im almost 24h with 6gpu! finally!

**Comment by mritten** [comment]
The only thing you did different was to go to a prior Windows 10 build?  The 1703?

**Comment by dragon9820** [comment]
Yes   Same afterburner, bios, miner configs,  drivers, and windows settings.

**Comment by Alexisgarci** [comment]
i went previous windows build, and i followed step by step the msi guide. i did exactly what it says to do.

**Comment by miningtech** [comment]
Been having exactly the same problems, with flickering display, TDR video errors, unable to create DAG on GPUx etc as soon as I installed the 6th card.  It would run fine with 5 but as soon as I added the 6th the problems would return.  The solution seems to be to manually set the virtual memory to at be at least 20gb (each DAG file is currently 2.3gb, so minimum of 13.8gb would be needed just for mining).  Windows was set to automatically manage mine and had settled at 12gb, I think Windows was trying to determine the size it needed itself, but was rejecting memory requests while it expanded the virtual memory, which would have resolved itself normally except every time this happened the machine crashed and restarted before it had time to increase the virtual memory allocation.

**Comment by Alexisgarci** [comment]
well my virtual memory is set to 5gb lol, i did not change anything except what i saw in the msi guide. Guess it worked fine for me! CLEAN windows buld at 1703, and follow step by step by step by step(lol) the guide. All working good with no errors no nothing.

**Comment by miningtech** [comment]
Possibly you have more physical memory installed than me (4gb) and hence need less virtual memory allocated.  Claymore's miner says on start up to ensure you have at least 16gb of virtual memory, so if problems re-occur as the DAG size grows then it may be worth checking this still.

**Comment by dragon9820** [comment]
Yea  virtual memory can definitely be a menace.  Glad you got it going!

**Comment by Alexisgarci** [comment]
Ye i have 32gb in it. Ill set it higher anyway, i haven't done it because it is still working.

**Comment by banshee1971** [comment]
For me i have 2 identifia rig, mine works fine with no issue. The other not working well (the one i try to fix)

Both have same config, except one got windows home (and always got latest windows update) and the other not stable has windows pro, All update disableUploadTimeout after the latest major update

**Comment by dragon9820** [comment]
I don't have any experience on Home,  but i'd check your version of pro,  type winver    If you're on 1709, I found it to be well worth formatting windows and reinstalling 1703, and don't install updates

**Comment by Alexisgarci** [comment]
Ye i’Ve disable auto updates via reg keys

**Comment by banshee1971** [comment]
The faulty RIG (Win PRO) : 1709  
My RIG (Home Ed) : 1709
Another PC i test with Home : 1709

I try the "Faulty RIG" with "SimpleMining" using the Linux boot from a USB key, got lot of issue again.

So now, i move 2 GPU on my "Another PC" that was doing nothing (PSU Corsair 750W) and i run the faulty RIG only with 2 GPU.

Something tell me the Southbridge CPU seem to have issue.

Using HMiNFO64, i install them on all PC, and monitor them from 1 PC.. I notice all the time, the faulty RIG have different PCIe Link Speed ... 
But my RIG : 8 GT/s ... on all 4 GPU ... 
So i run now a test... 48 hrs ! 2 GPU on the FaultyRIG (usually, works fine) and 2 other GPU on the other PC that was doing nothing !

Do you have the REG KEY for Disable Auto-update on Home Edition ?

On Pro, alreay disable them using the service.msc... not sure it will work on Home Edition

**Comment by banshee1971** [comment]
I just did it on My HomeEdition using Service.msc... will see.

**Comment by Alexisgarci** [comment]
when i installed the HWInfo i got diferent speeds on pcie to, now in this i didnt installed it yet cuz its working fine.

**Comment by Alexisgarci** [comment]
if you cant put yours to run properly, follow this https://www.msi.com/blog/Mining_Intel_Motherboard_Go_with_MSI_100_200_Series_Motherboard it worked for me running good for 4 or 5 days i think

**Comment by banshee1971** [comment]
In fact, i created my guide at the beginning, and it's the same as the one from MSI.
I test Simplemining that give me the same issue when using multiple card on that motherboard.

So i think the motherboard are faulty. I ask my friend to contact Amazon and ask for a replacement, since mine works fine with the same configuration (we have the same board, same gpu, memory, cpu. 

I check this morning his motherboard (i only plug 2 gpu) and my other old pc that running 2 of his card. No issues so far, 28Mhs stable per card.

**Comment by banshee1971** [comment]
Btw, i found this site very well explained : https://mining.help

**Comment by miewmiew333** [comment]
Guys, use iGPU to run the display. DO NOT plug in the the risers into the PCIE slot when installing windows. I prefer win10 which it'll automatic increase page files size, not sure about win7.

After installing windows and done all the updates, latest BIOS and settings.
1, Run Display Driver Uninstaller in safe mode, turn off the windows auto display driver installation (within the option)
2, Plug in ALL of the risers PCIE 1x, make sure power connected, turn on the second PSU if any first before turn on the motherboard.
3, Check Device Manager if all GPUs are being detected. If some GPUs not detected, identify which, swap Risers/USB 3.0 cable/PCIe to USB adapter. If you can see 4 out of 6 being detected, your BIOS settings are correct don't go dig around the BIOS settings already. 99% risers fault. BTW use GEN1 for faster boot up and will not affect hashrate.
4, Install display drivers. Software part complete. Here comes the fun part.
5, Start observe any weird problem like BSOD, freeze in blank screen when boot up, freeze when mining for minutes to hours. These freezing issue is caused by hardware. ALMOST all freezing issue is caused by bad risers, most of the time the USB 3.0 cable. If it freeze once, it'll freeze again, quickly swapped it out before it crash on you every time you went to sleep.
6, Check GPU Usage%, make sure it goes all the way to 99%-100%, if it keep dropping down to 80% or lower, identify that GPU, swap risers again until you get good straight line of GPU usage.

Bad overclocking only crash the mining software, very rare case like getting BSOD after hrs of mining. Very high CPU usage + bad hashrate is caused by display drivers, run DDU and install driver again.

I've installed many risers and rejected like about 20% of the risers, mostly USB cables, some PCIe 1x, only very few caused by the PCIe 16x socket. 180 risers, 30+ rejected. More GPUs coming next months.

**Comment by banshee1971** [comment]
I move GPU and their riser into 2 different motherboard. Run stable 27 Mhs for 1 days now.

And before that, i test 2 GPU (with their riser and cables) on the same motherboard for  48 hrs stability test each time, to see if i got 1 failed GPU or riser.

So in my case, what do you think?

**Comment by banshee1971** [comment]
I think it's the Southbridge that are defective. Can't handle more gpu. So the motherboard (because the chips are on the board).


Note : both MSI B250 board are configured like what describes in MSI site. I also purchase a Dummy HDMI to plug into the board, so i can use TeamViewer in high resolution!

**Comment by banshee1971** [comment]
My rig running stable for 3 weeks now! It's only the new one, that has issues. I was suspected faulty PSU, but when I shutdown my monero rig to move the PSU to this one, and notice no improvement... I start suspecting the motherboard.

When I open hwinfo64, and connect all rig to one master PC, i notice the PCIe speed issue on the faulty rig...

Memory error : 0! 
It's a freezing windows i got on the faulty rig, and freezing Claymore with Linux (after Claymore reboot after GPU error, it freeze after reboot) 
Tomorrow, i will be able to said that i have 48 hours stability...with the GPU and Riser.

**Comment by miewmiew333** [comment]
Definitely hardware issue, load optimize settings in the bios and redo the settings. Just to make sure no RAM configuration issue, or change the RAM to another slot and hope it's not defective RAM.

**Comment by miewmiew333** [comment]
I've experienced a bad PCI-e 1x slot in a motherboard before, getting detection issue and freeze during windows boot up.

**Comment by Yazdan-Ahmed** [comment]
I have been having this issue for some time now. After A LOT of troubleshooting, literally doing everything on every forum. 
PROCESS 
Things I tried
Update bios
4G encoding enables
PEG-> GEN 1
PEG -> IGD Display output from MB
PCI timing 32->96
Increase windows Virtual memory
Disable updates
Install older windows
Try multiple mining softwares including NH
Reinstall drivers
Try Simple miner OS
Running with HDD, and M.2.

Tested 4 different rigs, 3 have msi z-270 gaming M5 MB, and one ASUS H-270 prime
I have a 4 1070 ti rig running on the MSI board already for few weeks on windows. So i knew my rigs should work. But even after all the troubleshooting, still didnt get anywhere. 

So I figured since it had the same problem with SMOS and Windows, it has to be hardware. Cant be the CPU, MB, cause I know they work and whats the chances 4 of them go bad? I already tried M.2 vs. HDDD so that wasnt it. 

The only other difference between my working rigs and non working was the risers. I bought 60 risers from one guy and started using those. I was really doubting that all of them could be bad, and since I put new ones on a couple times, I assumed risers couldnt be it. 

SOLUTION
Well I decided to test risers further, I put two GPUs on the baord, and on another rig used 2 of a different type of riser. So far problem solved. Looks like it was the risers. 

MY recommendation FOR TROUBLESHOOTING
First, figure out if its hardware. Best way to do that, Try two different OS, Win.10 and Simple miner OS is probably your best bet. Of course make sure you set the bios to mining mode or 4G enabled. If it works on one OS and not the other, then you have an OS problem. If it doesnt work on either, then you most likely have a hardware problem. Then go form there. At the end of the day, finding out if its OS or HW first will save you a lot of time troubleshooting, especially if your problem is risers and you spend hours and hours troubleshooting OS... lol.

**Comment by Yazdan-Ahmed** [comment]
And yes the solution has been good for 24 hrs, the problem was that it would crash after 15 20 minutes. And then after restarting crash almost immediately (indicating a temp issue). I think a lot of people here are having issues with different causalities. The only solution the would work for everyone is a troubleshooting protocol that will definitely find the issue. 

My recommendation is first figure out if its hardware or software and double check Bios settings. 
Hardware troubleshooting is pretty common sense, unplug things, replug things, try replacement parts etc. 
OS troubleshooting might be a little more tedious, check windows event logs, try Virtual memory (not likely ever the issue since windows auto allocates), make sure youre watching temps, disable updates etc.

**Comment by banshee1971** [comment]
Already did that, and because windows was freezing.. There is no error in my case on the Event Log. But I remember that having trouble installing the network driver on the faulty rig, and not on the working rig. Both same brand and model.
So this ring me some bells. 
I'm contacting MSI and ask for replacement since that Motherboard are only 1 months old, and never worked correctly. All other items like GPU and riser worked perfectly on another rig. 

Simple mining OS shows me bettering the issue pn screen. I suspect the south

**Comment by banshee1971** [comment]
Still working with 2 GPU ! (on 2 differents MB). So My friends called Amazon (we purchase our board here), and we will receive a new board tomorrow !
I will then restart fresh, with this new MB !

**Comment by xixou2** [comment]
I also got rid of the crash by using a usb to rj45 adapter instead of the intel network chipset of the motherboard.

**Comment by dragon9820** [comment]
wow,  how did you discover that !!??

**Comment by xixou2** [comment]
Because I have another pc with a gtx1070 and a gtx1060 that never crashes on nicehash2.
I saw that my pc (4 gtx1070) crashed (froze) at the moment the connection to nicehash was lost for a while,
that was visible on the stable machine.
So then I borrowed a usb2.0 to ethernet 100 Mbps and my machine was stable for 1 week.
So I bought a usb3.0 to ethernet 1 Gpbs, but with that one the machine still freeze with ncehashv2 (no issue when I mine in direct way with ccminer).

So I'm now playing with other options of the usb adapter to see if that can help (in the device manager).

**Comment by miewmiew333** [comment]
Windows freeze definitely hardware issues, the only problem is identify which hardware, if you don't have many spare hardware on hand, that will be pain in the ass.

**Comment by xixou2** [comment]
I wen't back to my Intel motherboard network adapter (set to gigabit speed) and reduced speed of my DDR4, GTX 1070 and cpu core and nice hash is stable for last 12h.

**Comment by miewmiew333** [comment]
You can try WiFi adapter if you suspect wired ethernet hardware problem, it's pretty cheap, any branded low end will do.

**Comment by mritten** [comment]
It might be helpful to checkout your motherboard manufacturers site and download any new firmware drivers available for the chipset and Ethernet etc.

**Comment by banshee1971** [comment]
I received the new Motherboard, and same issue.

Hard to find when I have no spare part. 
I test all 4 GPU with their riser on my old computer, surprisingly works fine!

**Comment by banshee1971** [comment]
So now, only 2 items are not tested :
CPU and Memory. 

I i received a PC for one of my clients yesterday... So i now have access to a Spare memory module and CPU... I will test the easy one first... Memory.

**Comment by banshee1971** [comment]
Or probably tested the OS first (use Win10 home edition). I didn't tested the part yet.

**Comment by banshee1971** [comment]
I think tested the Home Edition will be my best bet !

All firmware are already at the latest version, and both motherboard has the same configuration ! I took picture of all page in my working RIG, and replicated the setting on the other rig. Both MB are the same !

The only difference, is the CPU (faulty RIG is Celeron G3930 Kaby Lake, and mine is G3900 skylake)
https://ark.intel.com/compare/97452,90741

**Comment by delicasu** [comment]
i have same problems here:
4x rx 570 stable, 6 rx 570 freeze.
first of all i want to change the risers from molex to pcie powered and see if the risers are the problem.
but i am not sure if i can do it in this way, please tell me your expirences:

i have 2x 700 watt psu. so 2x --> 4x 6+2pin connectors. 2 cables with 2 x 6+2pin.
on for MB+3x rx 570, the second only for additional 3x rx 570

can i power the gpu´s and pcie powered riser´s with 3x 8pin to 2x 6+2pin pci spiltter at the same time?
in this way:
1) connect 8pin splitter to the psu connector 3 times
2) connect 1 splitter end to the gpu with 6+2pin 3 times.
3) connect 1 splitter end to the riser with 6 pin 3 times.

so i have 3 of the 4 psu pci connections connected.
is there any problem with the wattage?

thank you very much

**Comment by miewmiew333** [comment]
You can power 2x RX570 with a single PCIE cable (two 8pin connector, one on each GPU), if the supplied cables are up to grade copper cable, if it's a CCA cable 18AWG like many PSU on the market, it'll run hot, but it'll still work.
The only limitation is the cable itself, those PCIE connectors can handle high current without issue.
I cannot fully answer your question because it all depend on the cable quality you're using, not the number of pins.
Molex 4pin powering riser work just fine, same goes with SATA. I'm using ver008s riser with multiple power input option.

**Comment by Alexisgarci** [comment]
yo dont use sata risers, sata dont pass the 75v that the riser may need. use 4pin risers and connect directly from psu to riser with no adapter!

**Comment by miewmiew333** [comment]
As I said, the power handling capabilities for each connectors are purely depend on the cable quality, the power rating is for the PSU manufacturer to follow so they don't cheap it out too much. An 8pin PCIE connectors can handle over 500w+ without issue if the wires are 16awg OFC, yes 10-15A per wire, but if it's made of 18awg tinned copper clad aluminium wire, it'll be just enough to meet the 150w rating, which a lot of PSU manufacturer do, because high quality copper wires are expensive.

**Comment by xixou2** [comment]
@Alexisgarci, 75W, not 75V
Try also to underclock the core and mem of your gpu to test the stability.

**Comment by xixou2** [comment]
My 5 gtx1070 rig on x399/1950x ran fine all night/day this time on nicehash 2.
I just had to underclock a bit the gpu/gddr5 mem with msi afterburner under win 10.

**Comment by Xinef87** [comment]
So team viewer was not the issue for me. I am now running 4x gtx 1070s without any issues. It is hardware issue for me, whether its the PCI slot on the mobo, the gpu itself or the riser, I have yet to determine. Will need to do some trial an error. I disabled the gpu that I found was causing this issue on Nicehash at the moment and its been running for over 24 hours now, without a freeze, crash of nicehash or pc restarting. This is with 80% power, +100 core, and + 600 memory clocks settings. As soon as I enable the 5th card, rig would eventually freeze or Nicehash would crash.

I know its that one card since I tried enabling that card, disabled another one on Nicehash and it froze the rig within 30 minutes. I know I swapped out all the risers previously as a step, so it maybe a faulty gpu. In the board explorer in the BIOS when I have all 5 cards connected, it detects it fine. But it is a step forward for me. Just some more trial and error.

**Comment by dragon9820** [comment]
i built another 6 gpu rig, i ended up having a bad gpu,  i plugged one in at a time and monitored the performance,  using stock settings in afterburner.  the first 2 were around 450 mh/s the 3rd was 230 mh/s and would crash excavator.  The other 3 were the same, around 450,  so on the 3rd gpu, i swapped risers, psu,cables, same result.  Been running 5 gpus the last 3 days with no issues  sending gpu back

**Comment by miewmiew333** [comment]
That's your display driver issue, you'll see high CPU usage all of the sudden and all kind of weird hash rate, GPU usage start to drop down, worse case if you have a weak CPU, it'll make the whole system very unresponsive that it felt like freezed.
Plug all the cards in run DDU, install the driver again with all GPU plugged in, don't do it one by one. Only do it one by one for troubleshooting GPU detection problem. It is best to install GPU with ethernet cable unplug because of windows automatic download and install the driver.

**Comment by btodd1** [comment]
Hi everyone. I came here because my rig started to lag and freeze but not completely crash. 
Window 10 PC w/1 EVGA GTX 1080 1000w PS. I've been running Nicehash for 3 weeks. No lag until the last 2 days. I read all the above and while doing so my Nicehash crashed and I accidentally discovered my problem to be very simple. I Hope this helps someone. I have two monitors, one is a 27" LG and the other a 23" LG. I have the 27" 75mhz using Display Port and the 23" 60mhz using HDMI (SAME CARD). I have been using my inferior monitor for my Nicehash Miner and Excavator to run and I can watch speeds while browsing on the left 27" monitor. Most of the time I don't use the computer with the miner running so it could have been an issue sooner. When my Nicehash miner stopping about 20 minutes ago, I moved it to the Superior monitor as well as the excavator. I was amazed to see the lag disappear immediately. I have been pushing my computer to crash while mining and no issues. If I take a window from the 27" to the 23" and move it around in circles, my GPU percent goes up to 68 to 80 percent. When I move it back it goes back down. I believe some of you may be having a display issue like me. Possible issues could be difference in MHZ, Drivers, or windows 10 trying to auto resize and connect my windows. I notice it's a struggle to move a window from one monitor to the other and when I do the window wants to stick in the middle. When I finally drag it over, it pops a bunch of windows together and I have to undo them. Anyway, not sure what the direct cause is but my computer is no longer lagging. It's been about 40 minutes now. I have not gone that long in the last two days! And I'm mining while typing this with no issues! Again, I hope this helps someone. Thanks  ;D

**Comment by miewmiew333** [comment]
Use integrated intel GPU for mining rig, the overclocking value is not stable for display graphic, which means you need to turn down the clock speed to get stability.
If you plan to use remote desktop, teamviewer or some other software to monitor your rig remotely, make sure you have a monitor plugged in to the iGPU or get a dummy HDMI for it, because once the rig restart without any display device connected, the iGPU will not be active and remote desktop will use discrete GPU to render graphics, which will cause the miner to crash if the overclocking is on its maximum level.

**Comment by Logman7585** [comment]
I've read through this thread and there's a lot of great ideas here. I've got a similar problem:

I have had 2x1080 and 2x1070 running perfectly stable. I put another 1080 and it will randomly fail. And by fail, I mean no blue screen, no driver crashes, just power turning off.  My instinct then is that it's a overcurrent protection. But I watch the loads on a Kill-a-watt and there's never a spike (I know the screen only refreshes once per second). With 5 cards the rig can run for anywhere from 5 minutes to 3 hours stable before a crash, but it's a hard power off when it happens. I did have that happen when I put 4 cards on one power supply and I know I exceeded the limit, but I don't think that's happening here (that said, I'm open to all ideas). Here's what I've done, all of which has left the same problem and had no effect:

New risers. Swapping in a different 1080 card from my gaming rig.  Using an empty riser in the 6th PCIe slot. Switch cards around to different orders and which order on the mobo the risers are connected. Switch which PSU is powering which cards (I have two 850w PSUs, and total power draw never visibly exceeds 1200w between the two or more than 700 on one) - 3 cards on one two on the other or vice versa. I've changed which PSU powers the mobo, and also shared the mobo between them (one on 24-pin, one on 4-pin, and molex, and all combinations of that). I can have all 5 installed and have only 4 of them mining and (Afterburner shows no useage on the 5th card) and it will still crash. That's why I'm doubting it's power usage. I'm using Nicehash, but Awesome Miner has the same problem. I've switched which card is off, turned off anywhere from all mining algorithms to just one on a card or up to two cards at the same time). Swapped the RAM for a new and larger RAM stick. Turned overclocks on, off, and had Afterburner off.

I'm open to ideas. The only thing I have left is yanking the 1000w out of my gaming computer and replacing one or a new motherboard? But each slot works perfectly fine by itself or with any other 3 slots, so I can't imagine it's a bad motherboard. Ideas?

**Comment by mritten** [comment]
Have you underpowered the cards using afterburner?  1070’s can usually be underpowered to around 75% and hash pretty well.  That may help you determine if it’s a power issue if the cards are drawing less power with afterburner running and under powering them all.

**Comment by mritten** [comment]
Also I’ve under powered 1080’a to around 55% without much hashrate drop.

**Comment by miewmiew333** [comment]
PSU trip definitely PSU under power problem. Let me guess, the PSU you're using is Coolermaster V-series? Cheap solution will be adding a cheap PSU like 400w-500w to power the 1080.

**Comment by Alexisgarci** [comment]
Bro if your system just crash like reboot, no blue screen no freeze it is a power issue. You tried a new psu more powerful? I use 2 1200w to power 6x 1080ti

**Comment by HodaCoder** [comment]
Same Problem here.
7x1080Ti Rig, Windows 10Pro utilizing EthControl mining Zcash via EWBF and Slightly under-powered. 
Crashing windows in 2 or 3 hours after mining start. 
This is what I have done upto now:
1- Apply https://github.com/DeadManWalkingTO/Windows10MiningTweaksDmW
2- Increase the Virtual Memory to the 30000MB
3- Disabling some of the monitoring polls of MSI AfterBurner as well as increasing the polling rate to 10000ms
Seems stable after 5 hours. Will update any change in status.

**Comment by btodd1** [comment]
I have an update to freezing windows while mining. I posted about using two different monitors and things got better when I changed to one. But I still had some lag although not as much. Also, like another post, I had no lag at all if I kept an internet window at full screen. Once I minimized, My computer froze. One time even had to hard boot as it would not respond. Solution for me: I have been reading a lot about tweaking my PC for mining. I tried lowering power on my XOC for both of my 1080 cards and it does seem to be more efficient. I also read something about down clocking my CPU. I have a 6700K (overclock) CPU. I am an enthusiast and have entered my PC in competitions so I'm familiar with overclock settings. My 6700K runs at 4.0GHZ standard. When I first started mining, I had it set to 4.4GHZ overclock and my RAM was set to XMI 30,000. I recently realized changing it back to stock speeds was when my lagging issues started. For me it seems to be all about what the CPU can do for other process on my PC during mining. (NOTE: I don't use my CPU for mining during these scenarios GPU ONLY). I was even gaming PUBG while mining before! I changed my overclock back to 4.3GHX and nearly eliminated my lag. A slight mouse stutter exists for 1 or 2 seconds then goes away. I can open and close windows on my desktop and no freezing. I changed clock again back to 4.4GHZ and no more stutter at all now. I know not everyone has K variant CPU's, but for those that do, this seems to be the ticket. I am no longer lagging or stuttering. I can browse and do things on my PC without issues. When not running anything, only mining,  on my PC when I'm away, I'm at 24C CPU and 48C GPU. Now while typing this I am remote VPN'd to my work computer and mining, my CPU is 31C. I do have water cooled CPU Corsair H115i and both GPU's are HYBRID water cooled. I hope this helps.

Thanks, Bret

**Comment by Logman7585** [comment]
Update: Great advice everyone. I underclocked all 5 GPUs to 75% and it's been stable for a day - not even that much of a drop in mining speed - 10% might be exaggerating. I guess I'm going to need some bigger PSUs before I add a 6th card. Thanks everyone for your help!

**Comment by Alexisgarci** [comment]
Ye bro major problems come from power issue. Like I said I use 2x1200w psu to run 6x1080ti. I have to say that I have 4x8gb ram, i7-7k water pump, risers etc. I know that 2400w is more then enough but at least I can secure when the cards have power spikes changing algos.

**Comment by cadenasv94** [comment]
Hi, I have a problem with my very strange mining rig.

the Gpu stop working suddenly without notifying any error in the pool or in the windonws. They just stop mining.

This error does not occur in certain time. Most of the time it is once in the day, but there are times that you work for two days well and then you stop working.

I have 6 GPU (4 EVGA GeForce GTX 1060 6GB and 2 MSI GAMING GeForce GTX 1060 6GB)

motherboard ASUS ROG STRIX B250H GAMING LGA1151 DDR4

4 Gb Ram

Intel BX80677G3930 7th Celeron Gene

60GB SSD

6 PCI-E Express Cable 1X TO 16X Graphics Extension Ethereum ETH Mining Powered Riser Adapter Card, 60cm USB 3.0 Cable, 4 Solid Capacitors (VER 008S)

Corsair RMi Series, RM1000i, 1000 Watt (1000W), Fully Modular Power Supply, 80+ Gold

The GPU is

Clock +100
Memory +750
Power limit 60
Fan speed +65

I use Claymore and nanopool 


What problem can it be?

**Comment by miewmiew333** [comment]
Unstable overclocking, not all 1060 cards stable running such high memory overclock. You can try push memory clock higher little by little and see which cards crash first, then declock that specific card to maybe +600 or lower. There are rigs that can be stable for 2 weeks and still crash/freeze due to aggressive tuning.

**Comment by xixou2** [comment]
Hi,
I noticed that freeze on my 5 GTX1070 and 1950X CPU is freezing on nicehash2 (equihqsh and cryptonightv7) when aida64 sensor panel is running.
Using nvidia drivers 391.35.
This is even worse with 397.64

I have opened a forum support ticket on aida64:
https://forums.aida64.com/topic/4220-aida64-sensor-panel-freezes-the-machine/

**Comment by ghost** [comment]
Remember with Nvidia cards to count the number of wires to gpu's and calculate the total amps these wires allow at one time all together, 
THEN, dont match wattage , check the amps your psu can deliver at max on the +12 volt. Make sure the overhead sits here in amps and not system, ram onboard gpu, cpu and those crap irrelevant considerations.  Its all about how much copper really.  You will find that you need at least more than double the watts to supply what your system can draw without having to cool things off from burning excessively .
 Matching tha watts only goes just about stable on fluctuating system demands (gaming) , not constant consistent amp draw from mining. The rule is one wire from psu per gpu. never split wires otherwise the cables supply non optimal and can burn from 3 or more nvidia gpu's on a single 4pin molex (11.5 amp max)   PSU cable, the weakest point being at your expensive psu for instance. Think power plant and  client , not aliminium smelter and generator  !

**Comment by Bluudeath** [comment]
New to mining, hope it's not to late. I have a 6x - RX580 - Win10, 2.9gb CPU, system. Nicehash Legacy will run for 5 minutes then quit, only doing Skunk mining. I have connected the AUXpwr1-2 for the risers, system keeps resetting. I disabled the Intel graphics card, system went crazy. Spent to much for a large paper weight, can any one assist. I think I am just change to Linux system and see if that helps, cards are VGA 8gb, pulling 150wats, using 1200wt PSU, is that to small?
